README.md

```diff
@@ -34,3 +34,22 @@ To clone and use this repository, you'll need Git Large File Storage (LFS).
 
 Our [Developer Guide](http://developer.lsst.io/en/latest/tools/git_lfs.html) explains how to setup Git LFS for LSST development.
 
+Usage
+-----
+
+<!-- TODO: replace with just links to Sphinx labels `ap-verify-datasets-install` and `ap-verify-running` once those docs are published -->
+
+This dataset must be cloned (with Git LFS) and set up with [EUPS](https://developer.lsst.io/stack/eups-tutorial.html) before it can be processed with `ap_verify`:
+
+    git clone https://github.com/lsst/ap_verify_hits2015/
+    setup -r ap_verify_hits2015
+
+Then, run `ap_verify` to process the dataset through the AP pipeline:
+
+    ap_verify.py --dataset HiTS2015 --id "visit=54123 ccdnum=25 filter=g" --output /my_output_dir/ --silent
+
+or run `ingest_dataset` to create standard Butler repositories for other purposes
```

Comment:
Consider saying that running `ap_verify` includes data ingestion, followed by ap_pipe stuff and metrics etc. As presented, a user could misconstrue that ingestion is handled entirely outside ap_verify, when in fact it happens as part of it.

Reply:
Sorry, I don't see how the current wording could be interpreted in any way other than `ap_verify` and `ingest_dataset` being alternatives. Besides, explaining `ap_verify` seems a bit out of scope.

Reply:
Not a big deal. Mostly I was concerned it could be seen as "either run ap_verify, or run ingest_dataset," but overall it is much improved, so thanks!

Reply:
I still don't get it. What's wrong with "either run ap_verify, or run ingest_dataset"? I thought your criticism was that it never makes sense to run ingest_dataset followed by ap_verify (which I think is true).

Reply:
Right; to me, the "or" implies you may do one or the other, and that they are independent options (one does not include the other as a step). We're probably both overthinking this.